config/samples/services_v1_auth.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: services.platform.opendatahub.io/v1
+kind: Auth
+metadata:
+  labels:
+  name: auth-sample
```

Comment:
according to the API, the name of the Auth CR is fixed to `auth`

Reply:
Yep, it was auto-generated and I missed it, fixed.